orcestra_book/plans/ATR-20240810a.md

```diff
@@ -48,15 +48,10 @@ categories: []
 ```{badges}
 ```
 
-## Crew
-
-The flight is planned to take off at {front}`takeoff` UTC.
-
-```{crew}
-```
-
 ## Flight plan
 
+* The flight is planned to take off at {front}`takeoff` UTC and land at {front}`landing` UTC.
```

Comment:
Are you takeoff/landing times actually given in UTC? I hope the handling will become easier once #76 is solved, but for now we should be clear on this.

Reply:
Hi Lukas, I can confirm that all the takeoff and landing times are given in UTC.